CMakeLists.txt

```diff
@@ -240,6 +234,7 @@ if (ENABLE_COVERAGE)
 endif()
 if (ENABLE_SANITIZER)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address")
+  set(OSRM_CFLAGS "${OSRM_CFLAGS} -fsanitize=address")
```

Comment:
Why do we expose the sanitizer flags to users? Is this needed to build the example?